components/centraldashboard/webpack.config.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+const { resolve } = require('path');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
```

Comment:
We're not using harmony import statements here?

I'm assuming this file also runs via babel?

Reply:
As a matter of fact it doesn't. It gets loaded and used from the webpack process with is running in the standard Node runtime which doesn't support ES6 out of the box.

https://nodejs.org/docs/latest-v10.x/api/esm.html